src/util/lang_string.ts

```diff
@@ -30,18 +30,50 @@ export const escapeXml = (string: string): string =>
  * @return {Array} array containing the graphemes
  */
 export const graphemeSplit = (textstring: string): string[] => {
-  const graphemes = [];
-  for (let i = 0, chr; i < textstring.length; i++) {
+  if ('Intl' in window && 'Segmenter' in Intl) {
```

Comment:
window is not going to work in node.
We have a special function to retrieve window in fabricjs and it is this one:
```
import { getFabricWindow } from '../env';
...
const window = getFabricWindow()
```

We may want to inspect later the difference in performance of initializing a new segmenter everyt time vs keeping it cached somewhere, but this is not important for this pr.